libs/hdf-converters/src/converters-from-hdf/splunk/reverse-splunk-mapper.ts

```diff
@@ -335,10 +365,19 @@ export class FromHDFToSplunkMapper extends FromAnyBaseConverter {
     });
 
     const uploads: Promise<AxiosResponse>[] = [];
+    logger?.info(
+      'Data converted, uploading to Splunk with config: ' +
+        JSON.stringify(config)
+    );
     uploads.push(...postDataToSplunkHEC(splunkData.reports, config));
     uploads.push(...postDataToSplunkHEC(splunkData.profiles, config));
     uploads.push(...postDataToSplunkHEC(splunkData.controls, config));
 
-    return Promise.all(uploads).then(() => guid);
+    return Promise.all(uploads).then(() => {
+      logger?.info(
```

Comment:
Is logger.info used during normal run-time? aka I don't have to specifiy `loglevel=info` ? 



@em-c-rod @Amndeep7 @camdenmoors 
I guess we should decide what is logged in:
Normal Mode
Debug Mode
Verbose mode - if we need it
`--quite` or `--silent` is the gnu standard to quite all stdout other than `exit 1/0`

Reply:
this comment will be talking more from the perspective of the saf cli than the library itself

presumably normal will just be states that we are passing through.  at minimum (and in most cases maximum), it will be 1) received input with x flags set, 2) processed input into hdf (or asff/xccdf/whatever), 3) (successfully or not) saved to file/uploaded to service the output.

debug mode would be making processes more visible.  i think io would fall in here - are we failing/succeeding (and if necessary how/why) with creating a file?  are we failing/succeeding (and if necessary how/why) with creating a connection to an external service?

verbose mode would be any data that is even more granular.  ex. if we decide to add any additional logging for where we are in the processing of an input file within hdf converters, it would go here (processed x/y findings from input into hdf/whatever) - personally I think this type of logging is useless except when trying to give the user a progress bar of some sort, which i think at the current speed of our mappers is not that useful since they usually complete within 30s at least with the possible inputs that i've seen thus far.

i don't think this pr can nor should resolve https://github.com/mitre/saf/issues/172

Reply:
The way this is setup is when this is run without a logging service or log level passed to it (e.g within Heimdall Lite), it will log debug messages by default, and would show inside of the browser console if opened. When done in the SAF CLI, we will allow the user to provide the option of what verbosity level they want to use, with a **default** of **Info**. All logging related to IO should be done within the application calling the mapper, not the mapper itself.